measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
```

Comment:
From discussion with @akegan I believe there are measure specifications for these code sets only for claims and registries. (see downloads from https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip)